usaspending_api/submissions/models.py

```diff
@@ -17,16 +17,17 @@ class SubmissionAttributes(models.Model):
     update_date = models.DateTimeField(auto_now=True, null=True)
 
     class Meta:
-        managed = True
         db_table = "submission_attributes"
 
     def __str__(self):
-        return "TOPTIER {} FY {} QTR {}".format(
-            self.toptier_code, self.reporting_fiscal_year, self.reporting_fiscal_quarter
+        return (
+            f"TOPTIER {self.toptier_code} "
+            f"FY {self.reporting_fiscal_year} "
+            f"QTR {self.reporting_fiscal_quarter} "
+            f"PRD {self.reporting_fiscal_period}"
         )
 
     @classmethod
-    def last_certified_fy(cls):
-        """FY for reporting purposes is the last FY for which submissions have been certified."""
-        result = cls.objects.filter(certified_date__isnull=False).aggregate(fy=models.Max("reporting_fiscal_year"))
+    def latest_available_fy(cls):
+        result = cls.objects.aggregate(fy=models.Max("reporting_fiscal_year"))
```

Comment:
The loader doesn't need to distinguish between certified and published. Really it just needs to know "published" since certification is a compliance aspect important in Broker but not shown in USAspending.

Reply:
Once the submission period table is in place, we will need to perform significant cleanup on several queries, this being one.

Having said that, though, I believe this new flavor of this function is 100% compatible with the old flavor.  Not saying it's correct, just saying nothing should have changed.  The other was filtering on certified only... but we only pulled certified so they all were being returned.  It was a meaningless filter.